docs/en/data_types/domains/IPv4.md

```diff
@@ -0,0 +1,179 @@
+## IPv4
+`IPv4` is a domain based on `UInt32` type, this is basically a typed replacement of `UInt32` for storing IPv4 values. It provides compact storage (compared to `String`), human-friendly output (compared to `UInt32`) and column type information on inspection.
+
+### Basic usage
```

Comment:
Title case for all English headers: Basic **U**sage
Here and in many places below

Reply:
Done